packages/core/core/src/Parcel.js

```diff
@@ -104,9 +108,9 @@ export default class Parcel {
     await initSourcemaps;
     await initRust?.();
     try {
-      initSentry?.();
+      initializeMonitoring?.();
```

Comment:
/nit I think a class instance would work better here, e.g.

```javascript
let monitoring = new ParcelMonitoring();

process.on('exit', () => {
  monitoring.close();
});
```

Reply:
This maps to a function so I don't think there's benefit to wrapping it in a class.

Reply:
I was also suggesting to change what it calls to a struct

Reply:
There's no reason to do that.